<ui version="4.0" >
 <class>luces</class>
 <widget class="QWidget" name="luces" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Llum:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" >
       <item>
        <property name="text" >
         <string>Llum Ambient</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 2</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 3</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 5</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 6</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Llum 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="boxAct" >
       <property name="text" >
        <string>Activa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="boxSel" >
       <property name="text" >
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>311</width>
       <height>281</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>311</width>
       <height>281</height>
      </size>
     </property>
     <property name="title" >
      <string>Color</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Ambient</string>
      </property>
      <widget class="QPushButton" name="BotonA" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>40</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Difus</string>
      </property>
      <widget class="QPushButton" name="BotonD" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>150</y>
        <width>121</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title" >
       <string>Especular</string>
      </property>
      <widget class="QPushButton" name="BotonE" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BotonC" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotonC</sender>
   <signal>clicked()</signal>
   <receiver>luces</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>306</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel" >
     <x>342</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
